thoth/package_extract/image.py

```diff
@@ -464,6 +465,17 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
+def _get_absolute_link(root_path: str, path: str, iter: int) -> str:
+    """Find the absolute link of the given link."""
+    if iter > _MAX_SYMLINKS:
```

Comment:
A warning here (we were not able to traverse symlinks)?

Reply:
Thanks, added a warning.

Reply:
Thanks, added